### ISSUE TITLE
ConfigReplacers scope clash with config import

### ISSUE THREAD

**Issue by neilstevenson** [hazelcast/hazelcast#15452]
A configuration replacers section this is imported does not replace variables in the file that imports it.

This fails to replace the group name,
```
<import resource="classpath:hazelcast3.xml"/>
<group>
    <name>$ENC{......}</name>
</group>           
```
If the imported file `hazelcast3.xml` contains a `<config-replacers>` section, the group name in the main XML file is not replaced.

Tested on 3.12.2, same behaviour with YAML as XML.

**Comment by neilstevenson** [comment]
And you might want different config replacers for Test v Production, so import the replacers file.

**Comment by alex-dukhno** [comment]
I'd like to try to work on the solution for the defect, if no one has started yet.

**Comment by alex-dukhno** [comment]
Seems like it could be closed as #15824 and #15810 are merged
Thank you

**Comment by mmedenjak** [comment]
Thank you for the contributions @alex-dukhno !
